CompressedImageLabel data
std_msgs/Header header